src/Spryker/Client/RabbitMq/Model/RabbitMqAdapter.php

```diff
@@ -7,13 +7,17 @@
 
 namespace Spryker\Client\RabbitMq\Model;
 
+use Generated\Shared\Transfer\QueueMetricsRequestTransfer;
+use Generated\Shared\Transfer\QueueMetricsResponseTransfer;
 use Generated\Shared\Transfer\QueueReceiveMessageTransfer;
 use Generated\Shared\Transfer\QueueSendMessageTransfer;
+use Spryker\Client\Queue\Model\Internal\MetricsInterface;
 use Spryker\Client\RabbitMq\Model\Consumer\ConsumerInterface;
 use Spryker\Client\RabbitMq\Model\Manager\ManagerInterface;
 use Spryker\Client\RabbitMq\Model\Publisher\PublisherInterface;
+use Spryker\Client\RabbitMq\Model\Queue\QueueMetricReaderInterface;
 
-class RabbitMqAdapter implements RabbitMqAdapterInterface
+class RabbitMqAdapter implements RabbitMqAdapterInterface, MetricsInterface
```

Comment:
It's direct dependency to Queue module.
First of all, if we go with this way, it should update required version in composer.

But do we really need this interface in the first place? Why we can't just provide data via some plugin and don't implement things for adapter?

Reply:
I mean, I get the idea that adapter is the thing that we can use in the Zed layer and there we add a plugin. But mb we can just introduce this interface right here?